app/javascript/components/side-panel.svelte

```diff
@@ -0,0 +1,100 @@
+<script context="module">
+  import gql from 'graphql-tag';
+  import Wifi, { fragments as WifiFragments } from '@components/wifi.svelte';
+  import Bathroom, { fragments as BathroomFragments } from '@components/bathroom.svelte';
+  import CarouselTab from '@components/carousel-tab.svelte';
+  import Logo from '@images/logo.png';
+
+  const getSidePanel = gql`
+    fragment getSidePanel on Location {
+      ...getWifi
+      ...getBathroom
+      timezone
+      widgets {
+        enabled {
+          id
+          sidebarText
+        }
+      }
+    }
+
+    ${WifiFragments.getWifi}
+    ${BathroomFragments.getBathroom}
+  `;
+
+  export const fragments = {
+    getSidePanel,
+  };
+</script>
+<script>
+  export let loading = false;
+  export let timezone;
+  export let widgets;
+
+  export let currentWidgetId;
+  export let showWeather;
+  export let durationSeconds;
+  export let isPaused;
+
+  $: enabledWidgets = widgets.enabled;
+
+  import Date from '@components/date.svelte';
+  import Time from '@components/time.svelte';
+  import filterFragments from '@lib/filter-fragments';
+</script>
+
+<style>
```

Comment:
I know it bucks the `svelte` convention, but have you tried putting `style` code at the bottom under the markup (`react-native` style)? I like it, but YMMV

Reply:
+1. Prettier has changed their default in the last few months from script-style-template to docblock-script-template-style.